### PR TITLE
Webserver

### PR DESCRIPTION
**Pull request by Shaktam**
#5

### CODE REVIEW

File: api-server/main.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import requests
+import json
+api_url_base = 'https://search.patentsview.org/api/v1/patent/10757852/'
+api_token = 'n8eiYbx8.oyBat24OZjTDA40SZ09tKrTvTiqgImye'
+
+headers = {'content-type' : 'application/json',
+'X-CSRFToken': 'zPZwOFHnynLs69FHTvhPeb868YPutic9hnTCz9WwK2HyIF0MMDAYkIJ9sUVa6juG',
+    'X-Api-Key': 'n8eiYbx8.oyBat24OZjTDA40SZ09tKrTvTiqgImye'}
+auth_response = requests.get(api_url_base, headers=headers)
```
Comment (irenepapaspyratos) [inline api-server/main.py:9]: I am not sure about your API, but when I find the values of TOKEN and KEY public readable, my hair raises...

File: infrastructure/autoscaling.tf [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  iam_instance_profile {
+    name = "LabInstanceProfile"
+  }
+
+  image_id = "ami-094125af156557ca2"
+  instance_type = "t3.micro"
+  key_name = "vockey"
+  vpc_security_group_ids = [aws_security_group.security_group.id]
+ # user_data = filebase64("script/userdata.sh")
```
Comment (irenepapaspyratos) [inline infrastructure/autoscaling.tf:19]: Usually you don't keep comments, that actually aren't real comments, in your code.
That's bad coding style - delete that.

**Review by irenepapaspyratos — COMMENTED** [review]
Only had a VERY quick look and wrote, what I noticed.

File: api-server/main.py (hunk 1 above)
Comment (fabianschmauder) [reply api-server/main.py:9]: Use env variables for this. now you keys are public. cloud be that you key gets blocked by the api now

**Review by Droggelbecher92 — APPROVED** [review]
LGTM :) Maybe would have waited with the Load-balancer, but seems right :)

**Pull request merged**